lxd/instances_post.go

```diff
@@ -652,6 +653,18 @@ func createFromBackup(s *state.State, r *http.Request, projectName string, data
 		bInfo.Name = instanceName
 	}
 
+	// Override the volume's UUID.
+	// Normally a volume (and its snapshots) gets a new UUID if their config doesn't already have
+	// a `volatile.uuid` field during creation of the volume's record in the DB.
+	// When importing a backup we have to ensure to not pass the backup volume's UUID when
+	// calling the actual backend functions for the target volume that perform some preliminary validation checks.
```

Comment:
This comment could be expanded to explain why "When importing a backup we have to ensure to not pass the backup volume's UUID"